### ISSUE TITLE
Does not create /etc/icinga2/features-available/ directory

### ISSUE THREAD

**Issue by TinajaLabs** [voxpupuli/puppet-icinga2#281]
This has started up.
```
Error: Could not set 'file' on ensure: No such file or directory @ dir_s_mkdir - /etc/icinga2/features-available/checker.conf20170418-15608-brelus.lock
```
* Redhat 6.7
* Current release of puppet-icinga
* It has worked OK on other hosts.

Applies to all features that have been defined and is obviously throwing the error when the features-available directory cannot be found.

Also noted that conf.d directory is not being written, although this might be because of the issues noted above.  I changed confd setting from false to true to 'confx.d'  - no change or effect.

Any tips would be appreciated.

Thanks,
Chris.

**Comment by lbetz** [comment]
Looks like the package isn't installed. Did you declare the main class icinga2 first? Or did you use manage_package => false and forgot to did it in front by your own?

**Comment by TinajaLabs** [comment]
I'll check that in the morning.  Thank you.

**Comment by bobapple** [comment]
@TinajaLabs do you have any feedback?

**Comment by TinajaLabs** [comment]
Yes.  I fixed it eventually by resetting the value of manage_package => true.

The problem is I had to completely delete all references to Icinga2 before it would go back to writing the directories.  It would be good to know what the rules are for creating the directories and which components, the existence of which would indicate the system is detected as previously installed.

I suspect that there are some files that are used to indicate that the system is already installed and so, won't create directories like conf.d.

Thanks for your help, bobapple.
